README.md

```diff
@@ -71,6 +71,17 @@ During a sprint we track progress using a [scrum or kanban-like board](#notes)<s
 
 After finishing a sprint we hold a retrospective to discuss what went well, what didn't go so well, and what we decide to do differently next sprint. Part of retrospective involves examining our [burndown chart](https://en.wikipedia.org/wiki/Burn_down_chart) to gauge the accuracy of our estimates and inform how we estimate and commit to the next sprint's tasks. Retrospective is also the perfect time to adjust the process to better suit our needs. 
 
+##Residencies
+
+In order to promote personal growth and inter-team knowledge transfer, the Mobile & TV Group offers residencies. A residency is a sprint-long period where anyone from any other group within Vimeo can join the team to work on any product that interests them. This allows the opportunity to learn new langagues and development skills.
```

Comment:
"This allows the opportunity to learn new langagues and development skills." ->

Is this sentence necessary since the first sentence in the paragraph says, "In order to promote personal growth and inter-team knowledge transfer, the Mobile & TV Group offers residencies."?

Consider removing or getting more specific. Something like, this is an opportunity to gain experience in new languages, new platforms, new architectural concepts, and new process practices.